src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,24 +1,14 @@
 databaseChangeLog:
-  - changeSet:
-      id: 1
-      author: sa
-      changes:
-        - createSequence:
-            sequenceName: id_seq
-            incrementBy: 1
-            startValue: 1
             
   - changeSet:
-      id: 2
+      id: "create sequence and table for process type"
```

Comment:
Ah, that was a miscommunication. You don't need to use numeric IDs for Liquibase changesets. The data in the tables should use UUIDs for their primary keys.